digest.go

```diff
@@ -68,9 +68,6 @@ var (
 	// ErrDigestInvalidFormat returned when digest format invalid.
 	ErrDigestInvalidFormat = fmt.Errorf("invalid checksum digest format")
 
-	// ErrDigestInvalidLength returned when digest has invalid length.
-	ErrDigestInvalidLength = fmt.Errorf("invalid checksum digest length")
```

Comment:
Do we need to keep this (and just deprecate it) to satisfy any backwards-compat requirements?